src/open_inwoner/pdc/tests/test_views.py

```diff
@@ -242,11 +242,11 @@ def test_category_detail_description_rendered(self):
         response = self.client.get(url)
 
         self.assertIn(
-            '<p class="p">A <em>descriptive</em> description</p>',
+            "A <em>descriptive</em> description</p>",
             response.rendered_content,
         )
         self.assertNotIn(
-            '[<p class="p">A <em>descriptive</em> description</p>, <em>descriptive</em>]',
+            "[A <em>descriptive</em> description</p>, <em>descriptive</em>]",
```

Comment:
We are going to have merge-conflict here since I've also changed these lines, but differently, in #1215 :-)

Reply:
We had a bug a while ago where CKeditor duplicated some of the content, so there's a reason whey the second assertion is in there